ui/shared/components/panel/variants/Pathogenicity.jsx

```diff
@@ -1,12 +1,23 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Label } from 'semantic-ui-react'
+import styled from 'styled-components'
+import { Label, Icon } from 'semantic-ui-react'
 
 import { CLINSIG_SEVERITY } from '../../../utils/constants'
 import { snakecaseToTitlecase } from '../../../utils/stringUtils'
 import { HorizontalSpacer } from '../../Spacers'
 
 
+const GrayStarIcon = styled(Icon).attrs({ name: 'star' })`
+  color: #D5D5D5;
+  margin: 0em 0.3em 0em 0em !important;
+`
+
+const YellowStarIcon = styled(Icon).attrs({ name: 'star' })`
+  color: #FFB70A;
+  margin: 0em 0.3em 0em 0em !important;
+`
```

Comment:
I would rather we just used semantic-ui's defaults for colors unless there is a strong reason to do otherwise

Reply:
Besides making it look better, idea was to match 
![image](https://user-images.githubusercontent.com/6240170/44169175-a13b5880-a0a1-11e8-90ce-e373cbbed98c.png)
(https://www.ncbi.nlm.nih.gov/clinvar/variation/263800/)
semantic ui grey doesn't work for this (can see an example here https://semantic-ui.com/usage/theming.html)
Especially for specialized components like this, seems better to treat semantic ui as a starting point instead of sticking to unmodified semantic ui built-ins.